EIPS/eip-6789.md

```diff
@@ -0,0 +1,88 @@
+---
+eip: 6789
+title: Rename `gas` to `mana`
+description: This EIP picks up on Vitalik Buterin's (@vbuterin) original proposal in 2015 to rename `gas` to `mana`
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/eip-6789-rename-gas-to-mana/13570
+status: Draft
+type: Standards Track
+category: Interface
+created: 2023-03-27
+---
+
+## Abstract
+
+This EIP picks up on Vitalik Buterin's (@vbuterin) original proposal in 2015 to rename `gas` to `mana`.
+
+## Motivation
```

Comment:
Additionally, you seem to have confused the Motivation and Rationale sections. The text you have here is great Rationale, and the text you have in the Rationale is great Motivation.